docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -41,7 +41,7 @@ The [example project](https://github.com/o1-labs/docs2/tree/main/examples/zkapps
 
 ## Prerequisites
 
-This tutorial has been verified with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.11.0` and [SnarkyJS](https://www.npmjs.com/package/snarkyjs) `0.12.1`.
+This tutorial has been verified with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.13.0` and [o1js](https://www.npmjs.com/package/o1js) `0.12.1`.
```

Comment:
if you have tested with 13.0, great! We have not officially tested with o1js, so the SnarkyJS release is accurate